ext/sockets/sockets.c

```diff
@@ -1811,6 +1811,7 @@ PHP_FUNCTION(socket_set_option)
 	HashTable		 		*opt_ht;
 	zval 					*l_onoff, *l_linger;
 	zval		 			*sec, *usec;
+	bool                                    failed;
```

Comment:
Bleh the indent doesn't match here. I would say to not bother, or remove the param name alignment.